### ISSUE TITLE
Python3 support

### ISSUE THREAD

**Issue by 2xyo** [fox-it/BloodHound.py#18]
Could you please support python3 as python2 is EOL in a few months?

`impacket` is not officially python3 compatible but an active python3.6 branch is available for testing, see https://github.com/SecureAuthCorp/impacket/issues/61#issuecomment-457389736

**Comment by dirkjanm** [comment]
Yes python 3 support is on the to-do, but so are many other things, so not sure when it will be ready.

**Comment by dirkjanm** [comment]
Python 3 support has been added in eff68f8. Turned out to be less work than expected. Only works with the latest impacket from master. Let me know if it works for you :)
